requirements/frozen.py3.txt

```diff
@@ -229,7 +229,7 @@ py==1.8.0
 pyasn1==0.4.5
 pyasn1-modules==0.2.4
 pycodestyle==2.5.0
-pycparser==2.19
+pycparser==2.17
```

Comment:
why downgrade ?

Reply:
This one was because of other package dependencies for pygit2, something else is requiring a version that is incompatible with pycparser>2.17.

I'm planning on updating our dependencies after this PR, with #9088.  So this is a temporary downgrade.